### PR TITLE
Deploy ceilometer-server role to SLE12 controller if there's one.

### PR DESCRIPTION
**Pull request by jsuchome**
Also deploy ceilometer-swift-proxy-middleware if swift is being deployed.

### CODE REVIEW

**Comment by rsalevsky** [comment]
+1

**Comment by jsuchome** [comment]
Not working yet, as mkcloud uses mongodb default and SLE12 mongodb is not yet on medias...

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -1747,6 +1747,10 @@ function custom_configuration()
             ;;
             ceilometer)
                 proposal_set_value ceilometer default "['deployment']['ceilometer']['elements']['ceilometer-cagent']" "['$sles12controller']"
+                proposal_set_value ceilometer default "['deployment']['ceilometer']['elements']['ceilometer-server']" "['$sles12controller']"
+                if [[ -n $deployswift ]] ; then
+                    proposal_set_value ceilometer default "['deployment']['ceilometer']['elements']['ceilometer-swift-proxy-middleware']" "['$sles12controller']"
```
Comment (vuntz) [inline scripts/qa_crowbarsetup.sh]: Isn't that done automatically? (ie, swift-proxy nodes automatically getting the role on proposal creation)

Comment (jsuchome) [inline scripts/qa_crowbarsetup.sh]: Hm, you're right, and swift controller (proxy) will already be at SLE12 node

**Comment by aplanas** [comment]
+1 here

**Comment by vuntz** [comment]
+1

**Pull request merged**